verl/trainer/config/ppo_trainer.yaml

```diff
@@ -82,7 +82,7 @@ actor_rollout_ref:
     log_prob_use_dynamic_bsz: ${actor_rollout_ref.actor.use_dynamic_bsz}
     log_prob_max_token_len_per_gpu: ${actor_rollout_ref.actor.ppo_max_token_len_per_gpu}
     disable_log_stats: True
-    enable_chunked_prefill: True # could get higher throughput
+    enable_chunked_prefill: False # may get higher throughput when set to True. When activated, Please increase max_num_batched_tokens or decrease max_model_len.
```

Comment:
Instead of setting this to False, is it better to add assertion in code to check max_num_batched_tokens and max_model_len

Reply:
```python
If max_num_batched_tokens > max_model_len and enable_chunked_prefill:
    raise ValueError(xxx)
```

Reply:
Sure